CLA.md

```diff
@@ -1,30 +1,66 @@
-### How do I sign the CNCF CLA?
+# The Contributor License Agreement
 
-* To sign up as an individual or as an employee of a signed organization, go to https://identity.linuxfoundation.org/projects/cncf
-* To sign up as an organization, go to https://identity.linuxfoundation.org/node/285/organization-signup
-* To review the CNCF CLA, go to https://github.com/cncf/cla
+The [Cloud Native Computing Foundation][CNCF] defines the legal status of the
+contributed code in a _Contributor License Agreement_ (CLA).
 
-***
+Only original source code from CLA signatories can be accepted into kubernetes.
 
-### After you select one of the options above, please follow the instructions below:
+This policy does not apply to [third_party] and [vendor].
 
-**Step 1**: You must sign in with GitHub.
+## Where do I sign?
 
-**Step 2**: If you are signing up as an employee, you must use your official person@organization.domain email address in the CNCF account registration page.
+Review the [CLA for individuals] and visit the [individual signup form] to sign
+as an individual or as an employee of a signed organization.
 
+To sign as an organization, read the [CLA for corporations] and visit [corp signup].
 
```

Comment:
Nit: I think the link to the sign-up page should be in the "How do I sign section"